target/linux/ramips/dts/mt7620a_alfa-network_r36m-e4g.dts

```diff
@@ -7,7 +7,7 @@
 
 / {
 	compatible = "alfa-network,r36m-e4g", "ralink,mt7620a-soc";
-	model = "ALFA Network R36M-E4G";
+	model = "BI4PCS-4G";
```

Comment:
So is it a new board? You should create a dedicated dts for it rather than hack the existing one.